### PR TITLE
fix: default to PUSH channel type for web analytics

### PR DESCRIPTION
**Pull request by hkjpotato**
#### Description of changes

Customers are still reporting the default endpoints Amplify created reaching the maximum limit, especially, during browser update.

The default endpoint we create does not have any channelType, and is used for collecting web analytics data. Currently, Pinpoint does not support a specific channel type for web analytics.

After discussion with service team, this change is to set the default channel type to "PUSH". This legacy channel type is not used for push notification, and will auto evict if reach maximum limits.

#### Issue #, if available

#7251
#6896

#### Description of how you validated changes
- localbrowser
- verify with service team

#### Checklist

- [x ] PR description included
- [x ] `yarn test` passes
- [ x] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by manueliglesias — APPROVED** [review]
LGTM 👍

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

File: packages/analytics/src/Providers/AWSPinpointProvider.ts [hunk 1]
```diff
@@ -659,7 +659,10 @@ export class AWSPinpointProvider implements AnalyticsProvider {
 			? clientInfo.platform === 'android'
 				? 'GCM'
 				: 'APNS'
-			: undefined;
+			: 'PUSH';
+		// NOTE: "PUSH" is a historical channel type, which is safe to be used for web analytics
```
Comment (calebpollman) [inline packages/analytics/src/Providers/AWSPinpointProvider.ts:663]: Does `event.address` always resolve as a truthy value in React Native usage? If not, using the verbiage of "web analytics" could be misleading here

Comment (hkjpotato) [reply packages/analytics/src/Providers/AWSPinpointProvider.ts:663]: sorry I miss the comment. will sync with you offline

**Pull request merged**